src/components/audio-call-game/game/sendingToServer/sendingToServer.ts

```diff
@@ -56,14 +56,17 @@ async function updateWordStatistic(correctAnswersList: WordWithUserWord[], wrong
 }
 
 async function updateGameStatistic(gameStatistics: GameStatistic, wordStatistics: WordsStatistic, correctAnswersList: WordWithUserWord[], wrongAnswersList: WordWithUserWord[], gameState: GameState, userStatistics: Statistic) {
+  const lengthAudiocallList = userStatistics.optional.audiocall.stat.length === 0 ? 0 : userStatistics.optional.audiocall.stat.length - 1;
```

Comment:
We can shorten lines length by making two additional variables and using them instead of the full object. Something like that:
```JS
const audiocallStat = userStatistics.optional.audiocall.stat;
const wordsStat = userStatistics.optional.words.stat;
```
Short lines would be easier to read and understand.

Reply:
good idea, commit - [13c6ee9](https://github.com/Jeneko/rslang/pull/114/commits/13c6ee9e8af42730db0629409aed91a2f086a79b)